inputfiles/patches/html.kdl

```diff
@@ -19,3 +19,13 @@ interface HTMLImageElement {
   property decoding overrideType=#""async" | "sync" | "auto""#
   property loading overrideType=#""eager" | "lazy""#
 }
+
+removals {
+  dictionary CanvasRenderingContext2DSettings {
+    member colorType // No implementation as of 2025-02
+  }
+
+  dictionary FocusOptions {
+    member focusVisible // Gecko only as of 2022-09
```

Comment:
Added in [WebKit in 2024](https://github.com/WebKit/WebKit/pull/32833) and [Blink a few weeks ago](https://issuetracker.google.com/462191849) 👀 Want to have another PR?